components/social/org.wso2.carbon.social.sql/src/main/java/org/wso2/carbon/social/sql/SQLActivity.java

```diff
@@ -109,4 +115,14 @@ public void setId(int id) {
 		JsonObject object = (JsonObject) this.body.get("object");
 		object.addProperty("id", id);
 	}
+
+	public void setRating(int rating) {
+		JsonObject object = (JsonObject) this.body.get("object");
+		object.addProperty("rating", rating);
+	}
+
+	public void setComment(String comment) {
```

Comment:
Thanks @thushara35 , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/9b346a71d58b4751d8306dcf4974e50076f97dd0)